docs/engage/front-end-ui/admin-section/admin-section.md

```diff
@@ -0,0 +1,169 @@
+---
+id: admin-section
+title: Admin Section
+sidebar_position: 1
+---
+
+The Admin Section provides a secure and permissioned interface for managing core operational and configuration aspects of the Engage platform. It enables authorized team members to carry out high-impact actions related to governance, token vesting, liquidity pool setup, and system permissions.
+
+This section is integral to maintaining the integrity of the ecosystem and ensuring that only designated actors can initiate or approve protocol-level changes. Role-based access is enforced through smart contract permissions, and all actions are executed transparently on-chain.
+
+The module is designed to support evolving platform needs, with extensibility for managing ecosystem wallets, approving proposals, or coordinating token distributions.
+
+---
+
+## 📌 Overview
+
+Defactor Engage provides a suite of powerful tools to administer and interact with the token economy. Admins can:
+
+- Manage token metrics and ecosystem wallets
+- Configure staking pools
+- Oversee governance proposals and votes
+- Monitor token buybacks
+- Maintain transparency with community stakeholders
+
+## Role-Based Access
+
+Only wallets designated as Admins can:
+
+- Access the Admin tab
+- Trigger system-level actions
+- View restricted system logs
+
+---
+
+## 🚀 Getting Started
+
+**Accessing the Admin Dashboard**
+1. Go to https://engage.defactor.com/dashboard
+2. Connect your wallet (MetaMask, Trust Wallet, or any WalletConnect-supported wallet)
+3. Ensure you're on the correct network (Polygon, Ethereum, Base, BNB Chain, or Algorand)
+4. Admin privileges will automatically unlock advanced configuration options if your wallet is whitelisted
+
+---
+
+## 📊 Token Metrics & Wallets
+
+**Token Supply & Distribution**
+
+Admins can monitor:
+
+- Total supply and circulating supply
+- Staked tokens and buyback totals
+- Network distribution (Ethereum, Polygon, etc.)
+> 📝 *Tip: Hover over tooltips for deeper insights into how these numbers are calculated.*
+
+**Ecosystem Wallet Management**
+
+Engage displays all project-owned wallets, including:
+
+- Wallet addresses
+- Live balances
+- Transaction history
+
+Admins can assign labels and explanations to each wallet for added clarity.
```

Comment:
the ecosystem wallets configured here will ne shown in the ecosystem dashboard , there is a default other wallet that will be automatically assigned any unallocated funds